### PR TITLE
Adds banners and robot showcase

### PR DESCRIPTION
**Pull request by DogZeroX**
## Screenshots 
image

:cl:
- add: Nanotrasen has purchased more decoration for the stations, banners, and some old robots they found somewhere deep in space-maint.
- tweak: The records computer no longer has a green puke color.
- add: Someone found an empty nuclear bomb and decided to be funny and fill it with cola. They are now dead and the cola is irradiated.

### CODE REVIEW

**Comment by DogZeroX** [comment]
image

**Comment by DogZeroX** [comment]
image

File: Resources/Prototypes/Entities/Structures/Machines/Computers/computers.yml [hunk 1]
```diff
@@ -104,7 +104,7 @@
     visuals:
     - type: ComputerVisualizer
       key: security_key
-      screen: security
+      screen: explosive
```
Comment (Elijahrane) [inline Resources/Prototypes/Entities/Structures/Machines/Computers/computers.yml:107]: What's going on with this change

Comment (DogZeroX) [reply Resources/Prototypes/Entities/Structures/Machines/Computers/computers.yml:107]: The records computer's layer is called explosive, I don't know why.

File: Resources/Prototypes/Entities/Structures/Decoration/banners.yml [hunk 2]
```diff
@@ -0,0 +1,136 @@
+- type: entity
+  id: BannerBase
+  parent: BaseStructureDynamic
+  name: base banner
+  description: It is the concept of a banner, you shouldn't be seeing this.
+  abstract: true
+  components:
+  - type: Sprite
+    sprite: Structures/Decoration/banner.rsi
+    state: banner
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Structures/Decoration/banners.yml:10]:
```suggestion
  - type: Sprite
    sprite: Structures/Decoration/banner.rsi
    state: banner
    netsync: false
```

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**